qt/applications/workbench/workbench/plotting/test/test_figureinteraction.py

```diff
@@ -731,7 +742,7 @@ def _create_mock_fig_manager_to_accept_right_click(self):
         fig_manager.canvas = canvas
         return fig_manager
 
-    def _create_mock_fig_manager_to_accept_middle_click(self):
+    def _create_mock_fig_manager_to_accept_middle_click(self) -> object:
```

Comment:
```suggestion
    def _create_mock_fig_manager_to_accept_middle_click(self):
```
I would put it back as it was, since specifying that `object` is returned is not very helpful, as everything in Python is an `object`